fda/grid.py

```diff
@@ -677,92 +685,157 @@ def concatenate(self, other):
                                                         other.data_matrix),
                                                        axis=0))
 
-    def _set_labels(self, ax):
+    def set_figure_and_axes(self):
+        """Set figure and its axes.
+
+        Returns:
+            fig (figure object): figure object initialiazed.
+            ax (axes object): axes of the initialized figure.
+
+        """
+        fig = plt.gcf()
+
+        if self.ndim_domain == 1:
+            projection = None
+        else:
+            projection = '3d'
+
+        ncols = math.ceil(math.sqrt(self.ndim_image))
```

Comment:
Why the sqrt? I think I am missing something.

Reply:
´ncols = math.ceil(math.sqrt(self.ndim_image))
nrows = math.ceil(self.ndim_image / ncols)´
The figure has as many axes as dimensions on the image. Imagine there are 4 dimensions,
we want a two graphs on top of another two, not 4 in line.

Reply:
I do not agree. I think that the disposition should be parametrizable, at least, even if we preserve the default of making square grids.

Reply:
changed